.github/workflows/release-draft.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+  # push: to remove above once test is ove
+  #   branches:
+  #     - develop
+  workflow_dispatch:
+    inputs:
+      FROM_TAG:
+        description: 'previous release tag version'
+        required: true
+        default: '5.0.0'
+      TO_TAG:
+        description: 'current release tag version'
+        default: 'HEAD'
+name: "Publish Release Draft notes"
+jobs:
+  preReleaseDraft:
+    name: Create pre-release draft
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - name: Install dependencies
+        run: yarn install
+      - name: Get from and to tags
+        id: tags
+        run: |
+          from_tag=$(echo ${{ github.event.inputs.FROM_TAG }})
+          to_tag=$(echo ${{ github.event.inputs.TO_TAG }})
+
+          if [ -z "$from_tag" ]
+          then
+                echo "::set-output name=from_tag::$(git describe --tags $(git rev-list --tags --max-count=1))"
+          else
+                echo "::set-output name=from_tag::$(echo $from_tag)"
+          fi
+
+          if [ -z "$to_tag" ]
+          then
+                echo "::set-output name=to_tag::HEAD"
+          else
+                echo "::set-output name=to_tag::$(echo $to_tag)"
+          fi
+      - name: Generate changelog
+        id: changelog
+        run: | 
+          changelog=$(npx ts-node scripts/changelog.ts --from user-4.1.0 --to ${{ steps.tags.outputs.to_tag }})
```

Comment:
hardcoded from for any below 4.2.0, as somehow there's no data for >= 4.2.0 

to be looked at